### DISCUSSION TITLE
Remove Time Component in Date Column

### DISCUSSION THREAD

**Discussion by leorae** [TiagoSilvaPereira/vemto-issues#227] [Q&A] [answered]
How do I achieve this through the app?

**Comment by TiagoSilvaPereira** [comment]
Hi @leorae could you please share more details?

**Reply by leorae** [reply]
I would like the dates to display as mmddyyyyy only so I select date as the option for the input field.  What shows up is mmddyyyy 0:00:00

**Comment by TiagoSilvaPereira** [comment, marked as answer]
Ahh ok, you can do it using a Laravel Accessor: https://laravel.com/docs/8.x/eloquent-mutators#accessors-and-mutators and using Carbon to format the date.

Or you can just use Laravel Date Casting: https://laravel.com/docs/8.x/eloquent-mutators#date-casting
